system/kube-monitoring/charts/prometheus-frontend/networking-dvs.alerts

```diff
@@ -0,0 +1,30 @@
+### Networking-DVS Alerts ###
+
+ALERT DVSPolling
+  IF min(irate(openstack_networking_dvs_plugins_ml2_drivers_mech_dvs_agent_dvs_agent_process_ports_timer_count[5m])) < 1
+  FOR 5m
+  LABELS {
+    service = "dvs",
+    severity = "critical",
+    context = "availability",
+    dashboard = "networking-dvs"
+  }
+  ANNOTATIONS {
+    summary = "Polling iterations fall below 1 per minute",
+    description = "Polling iterations of DVS Agent {{kubernetes_pod_name}} is below 1 per minute.",
```

Comment:
Access existing labels via `{{ $labels.kubernetes_pod_name }}` instead of `{{kubernetes_pod_name}}`.